tests/by-util/test_paste.rs

```diff
@@ -193,6 +193,27 @@ fn test_delimiter_list_ending_with_unescaped_backslash() {
     }
 }
 
+#[test]
+fn test_delimiter_list_empty() {
+    for st in ["-d", "--delimiters"] {
+        new_ucmd!()
+            .args(&[st, "", "-s", "--", "-"])
```

Comment:
I think `--` and `-` are not relevant for this test and so I suggest to remove them:
```suggestion
            .args(&[st, "", "-s"])
```